packages/template-react/src/context.js

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
I'd like to go without this file for now, especially, since we agreed, to remove graphql from the `template-react` package anyways, therefore, in future, we could just `import { createContext } from 'hops-redux';`. Could you inline this file in the `app.js`?

Reply:
Or maybe we can prepare a separate PR, that removes graphql from the default template and merge that PR, before we merge this.

Reply:
I think you are currently splitting the templates yourself, so yeah, let's do that,